core/geotime/src/main/java/mil/nga/giat/geowave/core/geotime/ingest/SpatialDimensionalityTypeProvider.java

```diff
@@ -261,4 +262,25 @@ else if (definition instanceof LongitudeDefinition) {
 		return hasLat && hasLon;
 	}
 
+	public static boolean isSpatial(
```

Comment:
can we just delegate isSpatial(PrimaryIndex) to call isSpatial(NumericIndexStrategy) with one extra null check for the index instead of duplicate code?